tests/Liuggio/StatsdClient/KeymetricTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Liuggio\StatsdClient;
+
+use Liuggio\StatsdClient\Sender\SysLogSender;
+use Liuggio\StatsdClient\StatsdClient,
+    Liuggio\StatsdClient\Factory\StatsdDataFactory,
+    Liuggio\StatsdClient\Sender\SocketSender,
+    Liuggio\StatsdClient\Service\StatsdService;
+
+class KeymetricTest extends \PHPUnit_Framework_TestCase
+{
+    private $sender;
+    private $client;
+    private $factory;
+
+    protected function setUp()
+    {
+        $this->sender = new SysLogSender();
+        $this->client = new StatsdClient($this->sender);
+        $this->factory = new StatsdDataFactory('\Liuggio\StatsdClient\Entity\StatsdData');
+    }
+
+    public function testMetrickey()
```

Comment:
Your test does not seem to test anything, you should at least give some asserts conditions.